test/gtest/common/test_perf.h

```diff
@@ -12,6 +12,10 @@
 #include <tools/perf/api/libperf.h>
 
 
+#define UCP_ARM_PERF_TEST_MULTIPLIER  2
+#define UCT_ARM_PERF_TEST_MULTIPLIER 15
+#define UCT_PERF_TEST_MULTIPLIER      5
```

Comment:
can we move these back to CC files?

Reply:
do you mean to return back the whole updates?
I just wanted to re-use code for all perf tests. and I guess it makes it simpler

Reply:
moved to test_perf.cc file